### ISSUE TITLE
wefwef.app fails Lighthouse and Pagespeed tests.

### ISSUE THREAD

**Issue by nathanielcwm** [aeharding/wefwef#155]
![image](https://github.com/aeharding/wefwef/assets/22572406/0dd01501-5fce-44fa-b9ae-fa0c0c9b4d50)

One of the recommendations is to enable compression for the Javascript. It appears like the API requests are already served with gzip - but I presume that it's because it's being pulled from lemmy.world which does have gzip compression enabled. [Nginx docs for gzip](https://docs.nginx.com/nginx/admin-guide/web-server/compression/), [Nginx docs for brotil](https://docs.nginx.com/nginx/admin-guide/dynamic-modules/brotli/)

**Comment by aeharding** [comment]
Hi, thanks!

The long term plan is that once CORS support is merged into lemmy, we can move wefwef out of an express server and onto static hosting (e.g. aws s3). Then we can enable code splitting since outdated chunks will be persisted (there was a bug related to this which is why I disabled code splitting).

TLDR initial content load is not a current priority, especially with service worker used for subsequent loads
